e2e/profile.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { test, expect, type Page } from '@playwright/test';
+
+const profilePageElements = {
+  camper: 'camper-profile',
+  portfolio: 'portfolio-projects'
+} as const;
+
+const certs = [
+  'Responsive Web Design',
+  'JavaScript Algorithms and Data Structures',
+  'Front End Development Libraries',
+  'Data Visualization',
+  'Relational Database',
+  'Back End Development and APIs',
+  'Quality Assurance',
+  'Scientific Computing with Python',
+  'Data Analysis with Python',
+  'Information Security',
+  'Machine Learning with Python',
+  'College Algebra with Python',
+  'Foundational C# with Microsoft'
+];
+
+const legacyCerts = [
+  'Front End',
+  'Back End',
+  'Data Visualization',
+  'Information Security and Quality Assurance',
+  'Full Stack'
+];
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/certifieduser');
+  await page.getByRole('button', { name: 'Preview custom 404 page' }).click();
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto('/certifieduser');
  await page.getByRole('button', { name: 'Preview custom 404 page' }).click();
});

test.afterEach(async ({ page }) => {
  await page.close();
});
```

Ideally we should run the tests in isolation.